### ISSUE TITLE
Please add sheets array to info

### ISSUE THREAD

**Issue by vimagick** [cristoper/gsheet#8]
This tool is very helpful. I want to automate a task by a URL like this:

```
https://docs.google.com/spreadsheets/d/{spreadsheetId}/edit#gid={sheetId}
                                        ^^^^^^^^^^^^^            ^^^^^^^
```

Then I'll export this sheet to csv:

```bash
$ gsheet info spreadsheetId
{
  "sheets": [...]
}

# find title by sheetId
$ jq '...'

$ gsheet csv --id spreadsheetId --range 'sheetTitle'
```

Thanks!

Ref:
- <https://developers.google.com/sheets/api/guides/concepts#sheet_id>
- <https://developers.google.com/sheets/api/reference/rest/v4/spreadsheets/sheets>

**Comment by cristoper** [comment]
Hi vimagick,

Are you able to try the [sheetinfo branch](https://github.com/cristoper/gsheet/tree/sheetinfo) (PR #9) and see if it works for your purposes? In particular the new `gsheet title --id $SHEET_ID --sheetid $GID` command in that PR should make it easy to look up a sheet title from a sheet id in your script.

**Comment by vimagick** [comment]
Great! 👏

**Comment by cristoper** [comment]
I merged the new commands to master and released v0.1.3:

https://github.com/cristoper/gsheet/releases/tag/v0.1.3
